easybuild/easyconfigs/w/Wannier90/Wannier90-2.0.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'Wannier90'
+version = '2.0.1'
+
+homepage = 'http://www.wannier.org'
+description = """A tool for obtaining maximally-localised Wannier functions"""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.wannier.org/code']
+
+# ugly hack
+prebuildopts = 'rm make.sys; touch make.sys; '
```

Comment:
well, this (and other) package don't have a configure script but instead expect you to alter a file (that is included in the Makefile) with the architecture dependent options. 

In this case I removed the default file and defined the options in the command line. I could have used something like sed to alter the file instead, but I don't know if that would be less ugly.

Maybe in these cases a different easyblock is needed to create/alter this file before building?